tests/core/Jalasoft.TeamUp.Projects.Core.Tests/ProjectsServiceTests.cs

```diff
@@ -132,5 +132,41 @@ public void DeleteProjectById_IdIsValid_ProjectRemoved()
             // Assert
             Assert.NotEqual(countBeforeDelete, countAfterDelete);
         }
+
+        [Fact]
+        public void UpdateProjectById_ProjectIsValid_ProjectUpdated()
+        {
+            // Arrange
+            var stubProject = new Project
```

Comment:
could this stub be abstracted to a method? so the unit test will be cleaner, please coul you try that?